e2e/test/iothub/service/RegistryE2ETests.cs

```diff
@@ -311,7 +311,7 @@ public async Task DevicesClient_RemoveDevices2Async_Works()
                 error1.And.StatusCode.Should().Be(HttpStatusCode.NotFound);
                 error1.And.ErrorCode.Should().Be(IotHubServiceErrorCode.DeviceNotFound);
 
-                Func<Task> act2 = async () => await serviceClient.Devices.GetAsync(device1.Id).ConfigureAwait(false);
+                Func<Task> act2 = async () => await serviceClient.Devices.GetAsync(device2.Id).ConfigureAwait(false);
```

Comment:
we already call `GetAsync` on device1 above (on L308), so correct this to device2 here